src/js/components/Drop/Drop.js

```diff
@@ -16,7 +16,19 @@ class Drop extends Component {
 
   originalFocusedElement = document.activeElement;
 
-  dropContainer = getNewContainer();
+  constructor(props) {
+    super(props);
+
+    this.dropContainer = getNewContainer();
+
+    // prevents mouse events from propagating upward to elements behind the drop.
+    // this prevents a nested drop from closing its parent when the user clicks
+    // inside it, since a 'nested' drop is technically an unrelated sibling in the
+    // DOM due to React portals.
+    this.dropContainer.addEventListener('mousedown', ev => {
+      ev.stopPropagation();
```

Comment:
Does this interfere with the ability to close the drop by clicking outside of it?

Reply:
Nope, this will basically stop events from bubbling upward from _inside_ the Drop. So any mousedown event which originates at a target within the DropContainer element will be prevented from reaching any of its parent elements, like `<document>`. Therefore, the handlers from other Drops which are listening for that event on `<document>` to close themselves will not be fired.

```
+----------------------+
| <document>           |
| +---------------+    |
| |<drop>         |    |
| |+------+       |    |
| ||button|=====>X|    | stops bubbling outside <drop>
| |+------+       |    |
| +---------------+    |
+----------------------+
```

Any mousedown event which originates outside the Drop would not trigger this handler, so propagation will not be affected and the event should reach the `<document>` and close all listening Drops.

```
+----------------------+
| <document>           |
| +---------------+    |
| |<drop>         |<========= calls handler
| |+------+       |    |   ||
| ||button|       |    |   ||
| |+------+       |    |   ||
| +---------------+    |   ||
|                      |   ||
| +------+             |   ||
| |<div> |============>| reaches document
| +------+             |
+----------------------+
```

Hope my diagrams make sense.

Reply:
At least, that's how I designed it. I'm still finding my way around your source code, so it's possible I attached the handler to the wrong DOM element.

Reply:
My intention is to attach it to the element which represents the visible, floating box. I think instead I may have attached it to the <div> added to <body> which helps position that box. I can do some more testing.